### PR TITLE
Dls 6718 packaging site details

### PR DESCRIPTION
**Pull request by GinnyThomas**
ITs are mainly shells as these will be done with navigation.  Notes in ticket for test coverage, how to reach page and test data to use.

### CODE REVIEW

File: app/models/UserAnswers.scala [hunk 1]
```diff
@@ -27,6 +28,7 @@ final case class UserAnswers(
                               id: String,
                               data: JsObject = Json.obj(),
                               smallProducerList: List[SmallProducer] = List.empty,
+                              packagingSiteList: List[Site] = List.empty,
```
Comment (Chrissystk) [inline app/models/UserAnswers.scala]: packagingSiteList with type List[Site] isn't very supportive for the update and delete elements to come in later stories. A better approach would be to use a Map[String, Site]. That way you can have a key for the Site being updated or removed. Unlike the smallProducerList which has a identifier within the SmallProducer class, the Site model doesn't. I started doing this with Jake so he could show you.

**Review by Chrissystk — DISMISSED** [review]
I still need to pull down locally to view, test etc.

File: app/models/UserAnswers.scala (hunk 1 above)
Comment (GinnyThomas) [reply app/models/UserAnswers.scala]: done

File: test/controllers/PackagingSiteDetailsControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        val result = route(application, request).value
+
+        val packagingSummaryList: List[SummaryListRow] =
+          PackagingSiteDetailsSummary.row2(Map.empty)(messages(application))
+
+        SummaryListViewModel(
+          rows = packagingSummaryList
+        )
+
+        application.injector.instanceOf[PackagingSiteDetailsView]
```
Comment (ChrisHillCap) [inline test/controllers/PackagingSiteDetailsControllerSpec.scala]: is this being used

**Review by ChrisHillCap — CHANGES_REQUESTED** [review]
1 comment, unless theres others like it and i missed it, minor, looks good

**Review by ChrisHillCap — APPROVED** [review]

**Pull request merged**